addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -324,6 +324,39 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
                     continue;
                 }
 
+                // Skip "animation-play-state" changes only
+                if (record.type === 'attributes' && record.attributeName === 'style') {
+                    const newValue = record.target.style.cssText;
+                    const oldAnimationState = (record.oldValue || '').match(/animation-play-state:\s*([a-z\-]+)/);
+                    const newAnimationState = newValue.match(/animation-play-state:\s*([a-z\-]+)/);
+                    if (oldAnimationState && newAnimationState && oldAnimationState[1] !== newAnimationState[1])
+                        continue;
```

Comment:
- Our convention is to always use `{``}` even if only one instruction
- Should not we `continue` if `oldAnimationState` or `newAnimationState` is truthy and not the other?